src/main/java/org/owasp/esapi/reference/DefaultEncoder.java

```diff
@@ -343,7 +344,18 @@ public String encodeForLDAP(String input, boolean encodeWildcards) {
                     sb.append("\\00");
                     break;
                 default:
-                    sb.append(c);
+                    if (c >= 0x80) {
+                        try {
+                            final byte[] u = String.valueOf(c).getBytes("UTF-8");
+                            for (byte b : u) {
+                                sb.append(String.format("\\%02x", b));
+                            }
+                        } catch (UnsupportedEncodingException ex) {
+                            // UTF-8 is always supported
```

Comment:
Suggest changing this to:
```java
    ;    // Ignore; UTF-8 is always supported
```
That way, it makes it a little more obvious of the intent when someone sees a\this (false positive) reported by a SAST tool. (Unfortunately, many of them still report this.)